src/view/ThemeManager.js

```diff
@@ -238,9 +238,12 @@ define(function (require, exports, module) {
                 })
                 .then(function (cssContent) {
                     $("body").toggleClass("dark", theme.dark);
+                    $("body").toggleClass("light");
                     styleNode.text(cssContent);
-
                     pending.resolve(theme);
+                    if($("body").hasClass("dark") === false && $("body").hasClass("light" ) === false){
```

Comment:
I think we should move away from body classes anyway and try it with an attribute, much cleaner.